languages/swift/exercises/concept/functions/.docs/after.md

```diff
@@ -0,0 +1,49 @@
+Great job! Functions are at the heart of Swift programming, so understanding them is vital to becoming fluent in Swift.
```

Comment:
The `after.md` file contains less examples than the `introduction.md` file, but that should probably be the other way around.